### DISCUSSION TITLE
Collection Run with Desktop App

### DISCUSSION THREAD

**Discussion by romil4** [hoppscotch/hoppscotch#4496] [Q&A] [answered]
Hi, 
With the use of Hoppscotch desktop app, is it possible to run a Collection? The only option I see is with the use of CLI which requires a self or cloud hosted Hoppscotch. Is there a way to perform the same with desktop app?

**Comment by SanskritiHarmukh** [comment, marked as answer]
Hi there, the GUI edition of Collection Runner is now available on all Hoppscotch platforms, including the Desktop app. Here's the [documentation](https://docs.hoppscotch.io/documentation/features/runner) and [video](https://youtu.be/mZvq4fMGw9Y) that you can check out to learn more.
